Digipost/APIClient.swift

```diff
@@ -433,9 +381,15 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                         self.deleteRefreshTokensAndLogoutUser()
                     }
             })
-        }else if (oAuthToken == nil) {
+        } else if (oAuthToken == nil) {
             assert(false," something wrong with oauth token")
```

Comment:
I've added more comments to explain whats going on. I'm postponing the actual fail harder until i've added the error handler with logging to our own service. 
commit: 225d5c6559219a2a2d776673eca5d5bb8f96d4dd